variables.yml

```diff
@@ -6483,6 +6483,26 @@ Price|Carbon|EU-wide Regulation All Sectors:
 Price|Carbon|EU-wide Regulation Non-ETS:
   definition: Price of carbon implied by EU-wide Non-ETS sector target
   unit: EUR2020/t CO2
+Price|Carbon|Final Energy|Transportation|Electricity:
```

Comment:
Good point. Thanks for noticing and questioning. Checking back with the team to confirm if these are the correct units.

Reply:
Unfortunately, we cannot say for sure if this is the correct variable, as we are currently having trouble finding out which team submitted the request for this variable to be added. We are working on improving the submission request to avoid "anonymous" variable change request. 

For now, please accept the addition as is. The unit can be interpreted as follows: price in EUR2020/GJ of final energy used for transport including the CO2 price. Meaning: the price includes CO2 tax (hence |Carbon) but is in EUR2020/GJ of final energy.